diff --git a/generated/ui_240x320/screens.c b/generated/ui_240x320/screens.c
index 6ec8e6b..36a22b7 100644
--- a/generated/ui_240x320/screens.c
+++ b/generated/ui_240x320/screens.c
@@ -659,24 +659,26 @@ void create_screen_main_screen() {
                             // HomeSignalLabel
                             lv_obj_t *obj = lv_label_create(parent_obj);
                             objects.home_signal_label = obj;
-                            lv_obj_set_pos(obj, 105, 50);
-                            lv_obj_set_size(obj, 78, 30);
+                            lv_obj_set_pos(obj, 42, 42);
+                            lv_obj_set_size(obj, LV_PCT(80), 30);
                             lv_label_set_long_mode(obj, LV_LABEL_LONG_CLIP);
                             lv_label_set_text(obj, _("LoRa RX"));
                             lv_obj_clear_flag(obj, LV_OBJ_FLAG_SCROLLABLE);
+                            lv_obj_set_style_align(obj, LV_ALIGN_LEFT_MID, LV_PART_MAIN | LV_STATE_DEFAULT);
                             lv_obj_set_style_text_line_space(obj, 0, LV_PART_MAIN | LV_STATE_DEFAULT);
+                            lv_obj_set_style_text_font(obj, &ui_font_montserrat_12, LV_PART_MAIN | LV_STATE_DEFAULT);
                             {
                                 lv_obj_t *parent_obj = obj;
                                 {
                                     // HomeSignalPctLabel
                                     lv_obj_t *obj = lv_label_create(parent_obj);
                                     objects.home_signal_pct_label = obj;
-                                    lv_obj_set_pos(obj, 0, 13);
-                                    lv_obj_set_size(obj, 78, 14);
+                                    lv_obj_set_pos(obj, 0, 12);
+                                    lv_obj_set_size(obj, 72, 12);
                                     lv_label_set_long_mode(obj, LV_LABEL_LONG_CLIP);
                                     lv_label_set_text(obj, _("idle"));
                                     lv_obj_clear_flag(obj, LV_OBJ_FLAG_SCROLLABLE);
                                     lv_obj_set_style_align(obj, LV_ALIGN_LEFT_MID, LV_PART_MAIN | LV_STATE_DEFAULT);
                                 }
                             }
                         }
diff --git a/generated/ui_320x240/screens.c b/generated/ui_320x240/screens.c
index 5d4dde2..664e064 100644
--- a/generated/ui_320x240/screens.c
+++ b/generated/ui_320x240/screens.c
@@ -533,24 +533,26 @@ void create_screen_main_screen() {
                             // HomeSignalLabel
                             lv_obj_t *obj = lv_label_create(parent_obj);
                             objects.home_signal_label = obj;
-                            lv_obj_set_pos(obj, 105, 50);
-                            lv_obj_set_size(obj, 78, 30);
+                            lv_obj_set_pos(obj, 42, 42);
+                            lv_obj_set_size(obj, LV_PCT(80), 30);
                             lv_label_set_long_mode(obj, LV_LABEL_LONG_CLIP);
                             lv_label_set_text(obj, _("LoRa RX"));
                             lv_obj_clear_flag(obj, LV_OBJ_FLAG_SCROLLABLE);
+                            lv_obj_set_style_align(obj, LV_ALIGN_LEFT_MID, LV_PART_MAIN | LV_STATE_DEFAULT);
                             lv_obj_set_style_text_line_space(obj, 0, LV_PART_MAIN | LV_STATE_DEFAULT);
+                            lv_obj_set_style_text_font(obj, &ui_font_montserrat_12, LV_PART_MAIN | LV_STATE_DEFAULT);
                             {
                                 lv_obj_t *parent_obj = obj;
                                 {
                                     // HomeSignalPctLabel
                                     lv_obj_t *obj = lv_label_create(parent_obj);
                                     objects.home_signal_pct_label = obj;
-                                    lv_obj_set_pos(obj, 0, 13);
-                                    lv_obj_set_size(obj, 78, 14);
+                                    lv_obj_set_pos(obj, 0, 12);
+                                    lv_obj_set_size(obj, 72, 12);
                                     lv_label_set_long_mode(obj, LV_LABEL_LONG_CLIP);
                                     lv_label_set_text(obj, _("idle"));
                                     lv_obj_clear_flag(obj, LV_OBJ_FLAG_SCROLLABLE);
                                     lv_obj_set_style_align(obj, LV_ALIGN_LEFT_MID, LV_PART_MAIN | LV_STATE_DEFAULT);
                                 }
                             }
                         }
